src/Icon/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import styles from './Icon.scss';
+import newId from '../utils/newId';
+
+function Icon(props) {
+  return (
+    <div>
+      <span
+        id={props.id ? props.id : newId('Icon')}
```

Comment:
Consider using `||` here instead - [`airbnb/javascript#15.7`](https://github.com/airbnb/javascript#comparison--unneeded-ternary)

Reply:
I went with the ternary based on the previous findings we had with `null/undefined` to ensure that cases were properly caught